deployer/hub.py

```diff
@@ -444,6 +461,7 @@ def deploy(self, auth_provider, secret_key, skip_hub_health_test=False):
                 # we should put the config from config/hubs last.
                 '-f', generated_values_file.name,
                 '-f', values_file.name,
+                '-f', secret_values_file.name,
```

Comment:
I see value in having this one separated from the main thing and applied at the helm step.
I think it is in agreement with the overall idea and the encrypted config pieces under secret.